toolkit/resources/manifests/package/pkggen_core_x86_64.txt

```diff
@@ -64,9 +64,9 @@ gettext-0.21-2.cm2.x86_64.rpm
 gzip-1.11-1.cm2.x86_64.rpm
 make-4.3-2.cm2.x86_64.rpm
 patch-2.7.6-7.cm2.x86_64.rpm
-util-linux-2.37.4-1.cm2.x86_64.rpm
-util-linux-devel-2.37.4-1.cm2.x86_64.rpm
-util-linux-libs-2.37.4-1.cm2.x86_64.rpm
+util-linux-2.37.4-2.cm2.x86_64.rpm
+util-linux-devel-2.37.4-2.cm2.x86_64.rpm
+util-linux-libs-2.37.4-2.cm2.x86_64.rpm
```

Comment:
If any of these packages require libraries from `libcap-ng`, you'll need to add `libcap-ng` to this manifest and update `update_manifests.sh` to put the install in the right place.